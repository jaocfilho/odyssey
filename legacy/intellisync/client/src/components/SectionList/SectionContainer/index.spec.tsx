import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import { SectionContainer } from '.';

describe('SectionContainer', () => {
  it('should render without crashing', () => {
    const { container } = render(<SectionContainer>children</SectionContainer>);

    expect(container).toMatchSnapshot();
  });
});
