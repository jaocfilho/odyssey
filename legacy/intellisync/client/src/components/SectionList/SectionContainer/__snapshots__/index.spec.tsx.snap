// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SectionContainer > should render without crashing 1`] = `
<div>
  <div
    class="grid max-w-7xl grid-cols-1 gap-x-8 gap-y-10 px-4 py-16 sm:px-6 md:grid-cols-3 lg:px-8"
  >
    children
  </div>
</div>
`;
